Fix HTTPS watermark

jeffkause opened the pull request:
## 🗣 Description ##

Lighten CISA watermark behind the report pages.

## 💭 Motivation and context ##

Fixes dark watermark issue in the HTTPS report Table of Contents.

![Screen Shot 2023-11-17 at 12 35 30 PM](https://github.com/cisagov/pshtt_reporter/assets/123761330/f992bf69-6244-4d26-a2e5-2a5eacab5ece)


## 🧪 Testing ##

Ran test report to verify the CISA watermark is transparent and the words on the pages can be read.


## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All new and existing tests pass.

## ✅ Post-merge checklist ##

 - [x] Deploy this change to production.

jsf9k commented on docker-compose.yml:
Please undo the changes to this file.

jsf9k commented on .DS_Store:
Please remove this file.

jsf9k commented on output/.DS_Store:
Please remove this file.

jsf9k commented on output/artifacts/.DS_Store:
Please remove this file.

jsf9k commented on output/artifacts/ocsp-crl.csv:
Please remove this file.

jsf9k commented on output/artifacts/reporting/.DS_Store:
Please remove this file.

jsf9k commented on output/artifacts/reporting/pshtt_reports/cyhy-USCCR-2023-11-17-https-report.pdf:
Please remove this file.

jsf9k commented on src/report.sh:
Please undo the changes to this file.

jsf9k commented on src/secrets/database_creds.yml:
Please undo the changes to this file.

jsf9k reviewed (changes requested):
You committed a bunch of files that you should not have.

jsf9k commented:
We will want a version bump for these changes as well.

jeffkause committed:
Reduce watermark opacity

Moving the watermark information from the file header to the Table of Contents, reduces the opacity of the watermark from 100% to 10%. Followed the changes from https://github.com/cisagov/trustymail_reporter/issues/54.

jsf9k committed:
Bump version from 1.2.13 to 1.2.14

jsf9k reviewed (approved):
Thanks for the contribution @jeffkause!

dav3r reviewed (approved):
👍 👍

jsf9k commented:
I updated production to use the Docker image from release 1.2.14 in production.  That is the version that will get used in next weekend's BOD 18-01 scanning run.
